images/capi/ansible/roles/providers/tasks/azure.yml

```
@@ -19,3 +19,19 @@
   vars:
     packages:
       - azure-cli
+
+- name: Ansible apt install chrony
+  apt:
+    name: chrony
+    state: present
+
+- name: fix chrony drift
+  shell: chronyc makestep 1.0 -1
+  args:
+    executable: /bin/bash
+
+- name: restart chrony.service to pick up config changes
+  systemd:
+    state: restarted
```

```suggestion
	enabled: yes
    state: restarted
```
Better to be explicit that we want it enabled on boot. The restart isn't technically necessary because of image creation.